testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -3,55 +3,54 @@
 
 ### This file contains the definitions for all steps concerning Cobbler.
 
+$cobbler_test = CobblerTest.new
+
 # cobbler daemon
 Given(/^cobblerd is running$/) do
-  ct = CobblerTest.new
-  raise 'cobblerd is not running' unless ct.running?
+  raise 'cobblerd is not running' unless $cobbler_test.running?
 end
 
 When(/^I restart cobbler on the server$/) do
   $server.run('systemctl restart cobblerd.service')
 end
 
+Given(/^I am logged into Cobbler API as user "([^"]*)" with password "([^"]*)"$/) do |user, pwd|
```

Comment:
```suggestion
Given(/^I am logged in via the Cobbler API as user "([^"]*)" with password "([^"]*)"$/) do |user, pwd|
```
I would use something like this.